src/main/charts/bitbucket/values.yaml

```diff
@@ -104,27 +104,32 @@ bitbucket:
 
   resources:
     jvm:
-      # -- The maximum amount of heap memory that will be used by the Bitbucket JVM. The same value will be used by the ElasticSearch JVM.
+      # -- JVM memory arguments below are based on the defaults defined for the Bitbucket docker container, see:
+      # https://bitbucket.org/atlassian-docker/docker-atlassian-bitbucket-server/src/master/
+      #
+      # -- The maximum amount of heap memory that will be used by the Bitbucket JVM. The same value will be used by the Elasticsearch JVM.
       maxHeap: "1g"
-      # -- The minimum amount of heap memory that will be used by the Bitbucket JVM. The same value will be used by the ElasticSearch JVM.
-      minHeap: "1g"
+      # -- The minimum amount of heap memory that will be used by the Bitbucket JVM. The same value will be used by the Elasticsearch JVM.
+      minHeap: "512m"
 
     # -- Specifies the standard Kubernetes resource requests and/or limits for the Bitbucket container.
     # It is important that if the memory resources are specified here, they must allow for the size of the Bitbucket JVM.
     # That means the maximum heap size, the reserved code cache size, plus other JVM overheads, must be accommodated.
-    # Allowing for (maxHeap+codeCache)*1.5 would be an example.
-    container: {}
+    # Allowing for maxHeap * 1.5 would be an example.
+    container:
     #  limits:
```

Comment:
Should we add some comment for the limits as well?

Reply:
I dont think its necessary we dont have one for `requests` and there will be detail for these 2 in the accompanying documentation: https://github.com/atlassian-labs/data-center-helm-charts/pull/137/files#diff-e0b8466f4f553e9957720b26bb44cad7e526abbd0950363ea570f76cd0517fff